--- cli.rb	2019-07-08 22:13:46.000000000 -0700
+++ cli2.rb	2019-07-09 10:27:50.000000000 -0700
@@ -96,7 +96,7 @@
     end

     def show_version
-      msg("#{ChefDK::Dist::PRODUCT} version: #{ChefDK::VERSION}")
+      msg("#{ChefDK::Dist::PRODUCT} version: $CHEF_WS_VERSION$")
       { "#{ChefDK::Dist::INFRA_CLIENT_PRODUCT}": "#{ChefDK::Dist::INFRA_CLIENT_CLI}",
         "#{ChefDK::Dist::INSPEC_PRODUCT}": "#{ChefDK::Dist::INSPEC_CLI}",
         "Test Kitchen": "kitchen",
