--- openssl-3.2.4/Configurations/windows-makefile.tmpl.org	2025-03-03 12:15:23
+++ openssl-3.2.4/Configurations/windows-makefile.tmpl	2025-03-03 12:23:04
@@ -455,7 +455,7 @@
 	@$(ECHO) "Tests are not supported with your chosen Configure options"
 	@{- output_on() if !$disabled{tests}; "\@rem" -}
 
-install: install_sw install_ssldirs {- "install_docs" if !$disabled{docs}; -} {- $disabled{fips} ? "" : "install_fips" -}
+install: install_sw install_ssldirs {- $disabled{fips} ? "" : "install_fips" -}
 
 uninstall: {- "uninstall_docs" if !$disabled{docs}; -} uninstall_sw {- $disabled{fips} ? "" : "uninstall_fips" -}
 
