diff --git a/apps/openssl-vms.cnf b/apps/openssl-vms.cnf
index ac858d6..d1cb967 100644
--- a/apps/openssl-vms.cnf
+++ b/apps/openssl-vms.cnf
@@ -48,17 +48,19 @@ tsa_policy3 = 1.2.3.4.5.7
 # fips provider. It contains a named section e.g. [fips_sect] which is
 # referenced from the [provider_sect] below.
 # Refer to the OpenSSL security policy for more information.
-# .include fipsmodule.cnf
+
+#  .include fipsmodule.cnf

 [openssl_init]
 providers = provider_sect

 # List of providers to load
 [provider_sect]
-default = default_sect
+base = base_sect
+legacy  = legacy_sect
 # The fips section name should match the section name inside the
 # included fipsmodule.cnf.
-# fips = fips_sect
+#  fips = fips_sect

 # If no providers are activated explicitly, the default one is activated implicitly.
 # See man 7 OSSL_PROVIDER-default for more details.
@@ -68,9 +70,11 @@ default = default_sect
 # becomes unavailable in openssl.  As a consequence applications depending on
 # OpenSSL may not work correctly which could lead to significant system
 # problems including inability to remotely access the system.
-[default_sect]
-# activate = 1
+[base_sect]
+activate = 1

+[legacy_sect]
+activate = 1

 ####################################################################
 [ ca ]
diff --git a/apps/openssl.cnf b/apps/openssl.cnf
index 12bc408..5b617b4 100644
--- a/apps/openssl.cnf
+++ b/apps/openssl.cnf
@@ -48,17 +48,19 @@ tsa_policy3 = 1.2.3.4.5.7
 # fips provider. It contains a named section e.g. [fips_sect] which is
 # referenced from the [provider_sect] below.
 # Refer to the OpenSSL security policy for more information.
-# .include fipsmodule.cnf
+
+#  .include fipsmodule.cnf

 [openssl_init]
 providers = provider_sect

 # List of providers to load
 [provider_sect]
-default = default_sect
+base = base_sect
+legacy  = legacy_sect
 # The fips section name should match the section name inside the
 # included fipsmodule.cnf.
-# fips = fips_sect
+#  fips = fips_sect

 # If no providers are activated explicitly, the default one is activated implicitly.
 # See man 7 OSSL_PROVIDER-default for more details.
@@ -68,9 +70,11 @@ default = default_sect
 # becomes unavailable in openssl.  As a consequence applications depending on
 # OpenSSL may not work correctly which could lead to significant system
 # problems including inability to remotely access the system.
-[default_sect]
-# activate = 1
+[base_sect]
+activate = 1

+[legacy_sect]
+activate = 1

 ####################################################################
 [ ca ]
